package.json

```diff
@@ -153,5 +157,13 @@
     "nyc": {
         "report-dir": "coverage/cypress/"
     },
-    "dependencies": {}
+    "packageManager": "pnpm@10.0.0+sha512.b8fef5494bd3fe4cbd4edabd0745df2ee5be3e4b0b8b08fa643aa3e4c6702ccc0f00d68fa8a8c9858a735a0032485a44990ed2810526c875e416f001b17df12b",
+    "pnpm": {
+        "onlyBuiltDependencies": [
```

Comment:
This flags packages that we explicitly allow to run `post-install` scripts during `pnpm install`. A nice safety feature to protect us from supply-chain attacks.